Files/Views/SettingsPages/Appearance.xaml

```diff
@@ -45,7 +45,7 @@
                         Grid.Column="0"
                         VerticalAlignment="Center"
                         FontSize="16"
-                        Text="Theme" />
+                        Text="Choose an app mode" />
```

Comment:
I would think the "Theme" is more descriptive, app mode can mean a number of other things such as tablet mode vs desktop mode...

Reply:
I am changing it to "App theme", feel free to revert this change if you prefer it the other way.

Reply:
Hi, this change was to follow Microsoft guidelines for App Settings

Reply:
@duke7553 Another Microsoft app
![image](https://user-images.githubusercontent.com/20501502/79241382-ab67b880-7e7b-11ea-99c3-9820c596972f.png)

Reply:
While following the Microsoft Guidelines are good, I don't see any reason to follow this one as it seems to make the description less clear.